README.md

```diff
@@ -500,19 +500,20 @@ Detailed specification is avaliable on zwave-ad repo under docs folder.
 
 Type | Interface                      | Value type | Description
 -----|--------------------------------|------------|------------------
-in   | cmd.usercode.clear             | str_map    |
-in   | cmd.usercode.clear_all         | null       |
-in   | cmd.usercode.get               | str_map    |
-in   | cmd.usercode.set               | str_map    |
+in   | cmd.usercode.clear             | str_map    | Delete specific slot
+in   | cmd.usercode.clear_all         | null       | Delete all users from all medium
+in   | cmd.usercode.get               | null       | Will return all slots for all medium
+in   | cmd.usercode.set               | str_map    | 
+in   | cmd.usercode.get_slot          | str_map    | Requesting specific slot {"slot": "3"}
 out  | evt.usercode.access_report     | str_map    |
 
+
 #### Service props
 
 Name             | Value example              | Description
 -----------------|----------------------------|-------------
-`sup_usercodes`  | ["pin", "rfid"]            | List of supported user code types
```

Comment:
I believe we should keep those. I checked zigbee-ad and it uses `code_type` (pin/rfid), `user_status` (enabled/disabled) and `user_type` (master/unrestricted). I think that's where those came from.